api/register/api.go

```diff
@@ -11,27 +12,28 @@ import (
 	"github.com/nyaruka/phonenumbers"
 )
 
+// ActionRepresentation struct
+type ActionRepresentation struct {
+	Name  *string `json:"name"`
+	Scope *string `json:"scope"`
+}
+
 // User representation
 type User struct {
+	UserName             *string `json:"userName,omitempty"`
```

Comment:
In some places you have UserName and in others Username (api/account/api.go)

Reply:
Kept Username